OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.h

```diff
@@ -0,0 +1,273 @@
+#ifndef OPENSIM_MOCOCONTACTIMPULSETRACKINGGOAL_H
```

Comment:
I'm not sure what happened -- I don't have any diffs for this file, and `MocoContactTrackingGoal.h` was deleted somehow. I'll do my best to review this file.